insecure/integration/functional/test/gossipsub/scoring/scoring_test.go

```diff
@@ -177,11 +177,18 @@ func testGossipSubInvalidMessageDeliveryScoring(t *testing.T, spamMsgFactory fun
 	blkTopicSnapshot, ok := topicsSnapshot[blockTopic.String()]
 	require.True(t, ok)
 
-	// ensure that the topic snapshot of the spammer contains a record of at least (40%) of the spam messages sent. The 40% is to account for the messages that were
-	// delivered before the score was updated, after the spammer is PRUNED, as well as to account for decay.
-	require.True(t, blkTopicSnapshot.InvalidMessageDeliveries > 0.4*float64(totalSpamMessages),
-		"invalid message deliveries must be greater than %f. invalid message deliveries: %f", 0.4*float64(totalSpamMessages),
-		blkTopicSnapshot.InvalidMessageDeliveries)
+	// none of the messages should be accepted by the victim node.
```

Comment:
this test became flaky as less than 40% of the messages were actually delivered. I'm guessing that the send logic is working faster now causing the spammer's queue to fill, dropping messages. What's important for the test is that  eventually the node disconnects from the spammer and stops receiving messages.